### ISSUE TITLE
`navigator.health` is undefined in capacitor 6. 

### ISSUE THREAD

**Issue by Saqib92** [dariosalvi78/cordova-plugin-health#352]
`navigator.health` is undefined on latest plugin version for capacitor. Followed all steps in docs to setup the plugin but its not working.

**Comment by dariosalvi78** [comment]
1. Do not use the old version, Google Fit API has been deprecated, possibly even discontinued.
2. navigator.health is not used any longer, we use cordova.plugins.health since version 3. Please pay attention at the README and / or the CHANGELOG

**Comment by Saqib92** [comment]
> 1. Do not use the old version, Google Fit API has been deprecated, possibly even discontinued.
> 2. navigator.health is not used any longer, we use cordova.plugins.health since version 3. Please pay attention at the README and / or the CHANGELOG

Yes, i was able to make it run with `cordova.plugins.health` and also not getting app block issue. Thank you. do i still need to add scopes in google cloud console and verify app?

**Comment by dariosalvi78** [comment]
no cloud console any longer, thanks God, see README

**Comment by Saqib92** [comment]
Thank you
